source/fundamentals/collations.txt

```diff
@@ -0,0 +1,280 @@
+   :language: javascript
+   :start-after: start findOneAndUpdate without collation
+   :end-before: end findOneAndUpdate without collation
+   :dedent: 4
+
+Since "Gunter" is the first result in the sorted results when using a binary
+collation, the operation does not update any documents. Consider the same
+operation with a collation specified with the locale set to
+``de@collation=phonebook``. This locale specifies the ``collation=phonebook``
+option which contains rules for prioritizing proper nouns. The
+``de@collation=phonebook`` locale and option sorts characters with umlauts
+before the same characters without umlauts.
+
+.. literalinclude:: /code-snippets/collation/findOneAndUpdate-collation.js
+   :language: javascript
+   :start-after: start findOneAndUpdate with collation
+   :end-before: end findOneAndUpdate with collation
+   :dedent: 4
+
+The ``findOneAndUpdate()`` operation above returns the following updated
+document:
+
+::
+
+  { lastErrorObject: { updatedExisting: true, n: 1 },
+    value: { _id: 3, first_name: 'Günter' },
+    ok: 1 }
+
+findOneAndDelete() Example
+``````````````````````````
+
+The following example calls the ``findOneAndDelete()`` operation on a
+collection named ``numbers`` which contains the following documents:
+
+::
+
+  { "_id" : 1, "a" : "16" }
+  { "_id" : 2, "a" : "84" }
+  { "_id" : 3, "a" : "179" }
+
+In this example, we set the ``numericOrdering`` collation parameter to ``true``
+to sort numeric strings based on their numerical order instead of their
+lexical order.
+
+.. literalinclude:: /code-snippets/collation/findOneAndDelete-collation.js
+   :language: javascript
+   :start-after: start findOneAndDelete example with collation
+   :end-before: end findOneAndDelete example with collation
+   :dedent: 4
+
+After you run the operation above, the collection contains the following
+documents:
+
+::
+
+  { "_id" : 1, "a" : "16" }
+  { "_id" : 2, "a" : "84" }
+
+If you perform the same operation without collation on the original
+collection of three documents, the server deletes the first document it
+finds in which the lexical value of "``a``" is greater than the lexical value
+of "``100``".
+
+.. literalinclude:: /code-snippets/collation/findOneAndDelete-collation.js
+   :language: javascript
+   :start-after: start findOneAndDelete example without collation
+   :end-before: end findOneAndDelete example without collation
+   :dedent: 4
+
+Since lexical value of all the values of ``a`` ("``16``", "``84``", and
+"``179``") are greater than the lexical value of "``100``", the operation
+removes the first result. After you run the operation above, the collection
+contains the following documents:
+
+::
+
+   { "_id" : 2, "a" : "84" }
+   { "_id" : 3, "a" : "179" }
+
+
+Aggregation Example
+```````````````````
+
+To use collation with the :node-api:`aggregate </Collection.html#aggregate>`
+operation, pass the collation document in the options field, after the
+array of pipeline stages.
```

Comment:
It might be helpful here to discuss how the collation applies to the aggregation pipeline. I'm assuming that every pipeline stage uses the specified collation, but if that's not true, we might want to point that out specifically here.

Reply:
I'm not absolutely certain when it applies in the aggregation pipeline, but I know that you can only specify one (and can include that in the docs if you think it's useful to provide that depth). Can you point me to any info on this? I checked the server docs and couldn't find anything.

Reply:
The docs aren't particularly clear (might be a good ticket to file for the manual) but I'd probably infer based on "You cannot specify multiple collations for an operation." and the fact that collation is a global option for all stages of an aggregation pipeline that the collation applies to all stages. Definitely an area where the docs could be more explicit!